Model/src/main/java/org/gusdb/wdk/model/WdkModel.java

```diff
@@ -653,6 +666,7 @@ public void close() {
     releaseDb(userDb);
     releaseDb(accountDb);
     Events.shutDown();
+    IOUtils.closeQuietly(managedCloseables.toArray(new Closeable[managedCloseables.size()]));
```

Comment:
I think we should use the FgpUtil version here.  I think we only have access to commons-io through a transitive dependency.  Could add but that is more work.  I find that if you use transitive deps with maven sometimes the build can break unexpectedly.

Reply:
Another option is to use FgpUtil's AutoCloseableList.  But then maybe all the things passed in must be autocloseable, not just closeable.